assets/components/directDebit/directDebitForm/sortCodeInput.jsx

```diff
@@ -6,17 +6,14 @@ import React from 'react';
 import type { SortCodeIndex } from 'components/directDebit/directDebitActions';
 
 type SortCodePropTypes = {
+  phase: string,
```

Comment:
`: Phase`?

Reply:
Good catch, thanks! I swear my IntelliJ search is not finding All The Things 👊